Sources/GraphQLTransportWS-DataSync/Server.swift

```diff
@@ -207,6 +208,8 @@ public class Server<InitPayload: Equatable & Codable> {
                 // swiftlint:disable:next force_cast
                 let stream = streamOpt as! ObservableSubscriptionEventStream
                 let observable = stream.observable
+                let disposeBag = DisposeBag()
+                disposeBags[id] = disposeBag
```

Comment:
nit: This isn't a bug you caused, but now that we're introducing state around the `id` values, we should probably check that the provided `id` is not already taken (that the client hasn't double-subscribed on a particular ID).

Reply:
Yes, it makes sense. I'll make the change.

Reply:
Excellent! Thanks!